code-scanning/properties/datree.properties.json

```
@@ -0,0 +1,7 @@
+{
+    "name": "Datree",
+    "creator": "Datree",
+    "description": "Detect misconfigurations in your Kubernetes manifests and present them in Github code scanning",
+    "iconName": "datree",
+    "categories": ["Code Scanning", "YAML"]
+}
```

```suggestion
}

```

To remove no new line at EOF error